GNUmakefile

```diff
@@ -41,7 +43,13 @@ test: fmtcheck
 		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
-	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
+	rm -rf $(PWD)/.terraform || true
+	mkdir $(PWD)/.terraform
+	mkdir -p /tmp/.terraform.d/localhost/test/kubernetes/9.9.9/$(OS_ARCH) || true
```

Comment:
Yeah, I really wanted to use a different directory. The problem I ran into is that during the terraform apply (inside the acceptance test), it was unable to find the relative path I gave it. So I had to choose a directory that I knew would be there on most systems (/tmp). I can experiment with others though and let you know.

Reply:
I see why I didn't do this. It's because I would need to hard-code the path in this file: .terraformrc

```
provider_installation {
  filesystem_mirror {
    path    = "/tmp/.terraform.d"
...
```

This is the file that acceptance tests read, and it doesn't seem to work with a relative path. So it looks like we're stuck with /tmp for now.

Reply:
Alright, sounds like this is the safest choice for now. If it runs well in TC, then let's go with it.